File: TheaterSuggestions/CSharp/ScreeningSeatingArrangementSuggestions/SuggestionsAreNotAvailable.cs
namespace SeatsSuggestions;

public class SuggestionsAreNotAvailable : SuggestionsAreMade
{
    public SuggestionsAreNotAvailable(string showId, int partyRequested)
        : base(showId, partyRequested, Array.Empty<SuggestionIsMade>())
    {
    }
}

File: TheaterSuggestions/Java/ScreeningSeatingArrangementSuggestions/src/main/java/org/weaveit/seatingplacesuggestions/SuggestionsAreNotAvailable.java
package org.weaveit.seatingplacesuggestions;

import java.util.List;

public class SuggestionsAreNotAvailable extends SuggestionsAreMade {

    public SuggestionsAreNotAvailable(String showId, int partyRequested) {
        super(showId, partyRequested, List.of());
    }
}
